health/egov-indexer/stock-indexer.yml

```
@@ -0,0 +1,66 @@
+      indexes:
+        - name: stock-reconciliation-index-v1
+          type: reconciliation
+          id: $.stockReconciliation.id
+          isBulk: true
+          jsonPath: $
+          customJsonMapping:
+            indexMapping: { "Data": { } }
+            fieldMapping:
+              - inJsonPath: $
+                outJsonPath: $.Data
+          timeStampField: $.stockReconciliation.auditDetails.createdTime
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)